ports/mimxrt/moduos.c

```diff
@@ -61,20 +61,34 @@ STATIC mp_obj_t os_uname(void) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_0(os_uname_obj, os_uname);
 
+static bool initialized = false;
+
+STATIC void TRNG_init(void) {
+    trng_config_t trngConfig;
+
+    TRNG_GetDefaultConfig(&trngConfig);
+    trngConfig.sampleMode = kTRNG_SampleModeVonNeumann;
+
+    TRNG_Init(TRNG, &trngConfig);
+    initialized = true;
+}
+
+uint32_t TRNG_random_u32(void) {
+    if (!initialized) {
```

Comment:
to avoid code duplication, this check for initialised could be moved into `TRNG_init`, and then the code here and below can just unconditionally call `TRNG_init` (or `trng_start`)

Reply:
I've changed that. It looks better. Initially I thought about that style, but rejected in in favor of saving a few clock cycles. Ok. It's just a call/return saved.